addon-test-support/@ember/test-helpers/dom/select.ts

```diff
@@ -0,0 +1,74 @@
+import getElement from './-get-element';
+import isSelectElement from './-is-select-element';
+import { __focus__ } from './focus';
+import settled from '../settled';
+import fireEvent from './fire-event';
+import { nextTickPromise } from '../-utils';
+import Target from './-target';
+
+/**
+  Set the `selected` property true for the provided option the target is a
+  select element (or set the select property true for multple options if the
+  multiple attribute is set true on the HTMLSelectElement) then trigger
+  `change` and `input` events on the specified target.
+
+  @public
+  @param {string|Element} target the element or selector for the select element
+  @param {string|string[]} options the value/values of the items to select
+  @return {Promise<void>} resolves when the application is settled
+
+  @example
+  <caption>
+    Emulating selecting an option or multiple options using `select`
+  </caption>
+
+  select('select', 'apple');
+
+  select('select', ['apple', 'orange']);
+*/
+export default function select(target: Target, options: string | string[]): Promise<void> {
+  return nextTickPromise().then(() => {
+    if (!target) {
+      throw new Error('Must pass an element or selector to `select`.');
+    }
+
+    if (typeof options === 'undefined' || options === null) {
+      throw new Error('Must provide an `option` or `options` to select when calling `select`.');
+    }
+
+    const element = getElement(target) as any;
```

Comment:
why do we cast this to `any` here?

Reply:
I used the cast as any because it can return `Element | Document | null` also this is also how it is handled in fillIn so thought it would be best to keep the same for clarity

Reply:
`null` will be caught by the following condition, and `isSelectElement()` should automatically cast it down to a `HTMLSelectElement`